packages/use-sync-external-store/src/__tests__/useSyncExternalStoreShared-test.js

```diff
@@ -899,7 +899,7 @@ describe('Shared useSyncExternalStore behavior (shim and built-in)', () => {
         store.set({});
       });
       expect(container.textContent).toEqual(
-        "Cannot read property 'toUpperCase' of undefined",
```

Comment:
## before

<img width="1064" alt="Screenshot 2022-02-06 at 22 07 17" src="https://user-images.githubusercontent.com/2394070/152863804-ab0d9b29-72ab-4d94-acec-5357b98bd2f5.png">


TypeError message is changed from node.js 16